Source/Core/Core/HotkeyManager.h

```diff
@@ -34,6 +34,7 @@ namespace HotkeyManagerEmu
 	void Shutdown();
 
 	InputConfig* GetConfig();
-	void GetStatus(u8 _port, HotkeyStatus* _pKeyboardStatus);
+	void GetStatus();
+	bool OwnsLock();
```

Comment:
Mostly a matter of taste, but does the outside need to know about the locking going on? Maybe rename it to ```IsReady```/```IsBusy```` or something like that instead? Fine to ignore tho.